.github/workflows/coverage.yml

```diff
@@ -35,6 +35,10 @@ jobs:
       - uses: taiki-e/install-action@cargo-llvm-cov
       - uses: taiki-e/install-action@nextest
 
+      - name: Getting neccesary files for testing
+        run: |
+          wget -P ./crates/prover-services/sharp-service/tests/artifacts https://madara-orchestrator-sharp-pie.s3.amazonaws.com/238996-SN.zip
```

Comment:
so to run tests locally we need this right ? is that added into some readme or smth ? could probably be downloaded right in code too

Reply:
I will add this in readme that this file is needed for testing.